CopilotKit/packages/backend/src/lib/copilotkit-backend-implementation.ts

```diff
@@ -89,25 +89,18 @@ export class CopilotBackendImplementation {
       }
     }
 
-    // merge server side functions with langserve functions
-    let mergedTools = mergeServerSideTools(
-      this.actions.map(actionToChatCompletionFunction),
-      langserveFunctions.map(actionToChatCompletionFunction),
-    );
-
-    // merge with client side functions
-    mergedTools = mergeServerSideTools(mergedTools, forwardedProps.tools);
+    const serversideTools: Action<any>[] = [...this.actions, ...langserveFunctions];
```

Comment:
we need to pass `serversideTools` to `copilotkitStreamInterceptor` below.
So we are defining it as a single unit once explicitly as `serversideTools`, instead of twice implicitly (Before, we were calling `[...this.actions, ...langserveFunctions]` again)